tests/percy/__tests__/marty.js

```diff
@@ -85,15 +85,11 @@ describe('Marty Member user test', () => {
       expect(text_content).toEqual(expect.stringContaining('Body content'));
       expect(text_content).toEqual(expect.stringContaining('URL redirects'));
 
-      await page.type('#edit-title-0-value', 'Test page');
-      await page.type('#edit-field-summary-0-value', 'Summary for the test page');
-      await page.type('#edit-field-menu-link-text-0-value', 'Test page');
-      await page.evaluate(() => {
-        ck5 = Drupal.CKEditor5Instances.keys().next().value;
```

Comment:
Can these changes be reverted since the CKEditor bug was fixed, or is there another reason for them?

Reply:
The locator() would be a much better way to interact with the page. I only updated this test while trying to debug the failed test and didn't have time to update all tests yet. Let me add a future story.

Reply:
Created PGOV-1389 to update all tests.